src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -1933,8 +1933,8 @@ void EIRFuelFiredHeatPump::processInputForEIRPLHP(EnergyPlusData &state)
             // A4
             std::string sourceSideInletNodeName = UtilityRoutines::MakeUPPERCase(fields.at("air_source_node_name").get<std::string>());
             // UtilityRoutines::MakeUPPERCase(fields.at("source_side_outlet_node_name").get<std::string>());
-            srand(time(NULL));
-            std::string sourceSideOutletNodeName = format("DUMMY_CONDENSER_{}_{}", rand(), rand());
+            // srand(time(NULL));
+            std::string sourceSideOutletNodeName = "DUMMY_CONDENSER_OUTLET_NODE"; // format("DUMMY_CONDENSER_{}_{}", rand(), rand());
```

Comment:
This shouldn't use the same name for all instances of this heat pump. I would use the `thisPLHP.name + "Source Side Outlet Node"`. (It's not a condenser when it's in heating mode).

Reply:
Thanks! Changed applied. So, also, should the source outlets have different names for the heating hp and the cooling hp in case of a pair of companion coils?

Reply:
The PLHP's should have different names in that case, so it should be ok. But I'm not sure if unique plant component names are enforced except within a given object class.